web/package.json

```diff
@@ -69,8 +69,8 @@
     "redux-thunk": "^2.3.0",
     "styled-components": "^5.1.1",
     "translations": "0.0.1",
-    "url-polyfill": "^1.1.5",
-    "whatwg-fetch": "^3.0.0"
+    "url-polyfill": "^1.1.12",
```

Comment:
What do we need this for? IE?

Reply:
To be honest - no idea. But I just had a look into the git history:
![image](https://user-images.githubusercontent.com/43984911/106399087-eb058e00-6416-11eb-8413-2635d5e4269d.png)

Reply:
Haha yeah, good find :+1: